luci-app-passwall/Makefile

```diff
@@ -35,7 +35,7 @@ PKG_CONFIG_DEPENDS:= \
 LUCI_TITLE:=LuCI support for PassWall
 LUCI_PKGARCH:=all
 LUCI_DEPENDS:=+coreutils +coreutils-base64 +coreutils-nohup +curl \
-	+chinadns-ng +dns2socks +dns2tcp +ip-full +libuci-lua +lua +luci-compat +luci-lib-jsonc \
+	+chinadns-ng +dns2socks +dns2tcp +ipt2socks +ip-full +libuci-lua +lua +luci-compat +luci-lib-jsonc \
 	+microsocks +resolveip +tcping +unzip \
```

Comment:
should be add like `+PACKAGE_$(PKG_NAME)_INCLUDE_tuic_client:ipt2socks`
giving a chance to choose

Reply:
https://github.com/xiaorouji/openwrt-passwall/pull/2710/files